product_docs/docs/pem/9/pem_rel_notes/970_rel_notes.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Postgres Enterprise Manager 9.7.0 release notes"
+navTitle: Version 9.7.0
+---
+
+Released: 13th Aug 2024
+
+New features, enhancements, bug fixes, and other changes in PEM 9.7.0 include:
+
+| Type         | Description                                                                                                                                                                                                                                  |
+|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhancement  | Added PGD extension details in the Core usage report.                                                                                                                                                                                |
+| Enhancement  | If max_connections is set to 1 in the `agent.cfg` file, PEM agent startup process makes it to 2 in the backend code to avoid PEM's probe data collection failure. [Support |
```

Comment:
What is this string "[Support |" at the end of this note

Reply:
updated